segmentation/utils/data_utils_test.py

```diff
@@ -22,6 +23,22 @@ def _create_img_dir(temp_dir, fovs, imgs, img_sub_folder="TIFs"):
             io.imsave(os.path.join(fov_path, img), tif)
 
 
+def test_load_imgs_from_mibitiff():
+    mibitiff_files = [os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                   "..", "..", "data", "example_dataset",
+                                   "input_data", "input_data_TIFF",
+                                   "Point8_RowNumber0_Depth_Profile0-MassCorrected-Filtered.tiff")]
+    channels = ["HH3", "Membrane"]
```

Comment:
Adding a channels=None test case should catch the error in the main function

Reply:
Done.